### DISCUSSION TITLE
Página de vacunación 95%

### DISCUSSION THREAD

**Discussion by DanielJuarezH** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#342] [Show and tell]
Hola!

Les comparto mi trabajo con la página de vacunación, creo que para ser mi primer página no quedo mal. Aun estoy trabajando en la barra de navegación para agregar efecto al background y en los triángulos a lo largo de la página.

Se aceptan críticas, preguntas y/o comentarios :)

<a href="https://danieljuarezh.github.io/Vacunation-website/">Este es el link a mi página</a>

**Comment by Pablobeg** [comment]
que buena pagina bro 👌👌🙌, ¿como hiciste para ponerla en linea con github?

**Reply by DanielJuarezH** [reply]
Solo seguí el tutorial de GitHub pages, está muy bien explicado. Inténtale, si te atoras me dices y te ayudo. <br>
<a href="https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site">Link al manual de GitHub Pages</a>

**Comment by SujeiRodriguez** [comment]
:0 yo quiero hacer ese efecto en los rectángulos... a ver si puedo :p

oigan ustedes también tienen una biblia de css?, necesito saber limpiar el código o no escribir tanto, creo que repito mucho

**Reply by DanielJuarezH** [reply]
Te refieres a las sombras o a que cambien de color cuando pasas el mouse? <br>
Si es para las sombras, te dejo las dos páginas de las que yo aprendí a hacerlo:  <a href="https://www.w3schools.com/css/css3_shadows.asp"> link a w3schools</a> y <a href="https://css-tricks.com/almanac/properties/b/box-shadow/"> link a CSS-Tricks</a><br>
Si es para que cambien cuando pases el mouse, estas son las páginas: <a href="https://www.w3schools.com/csSref/sel_hover.asp">link a w3schools</a>, <a href="https://css-tricks.com/almanac/selectors/h/hover/">link a CSS-Tricks</a> y <a href="https://css-tricks.com/css-link-hover-effects/">aquí</a> algunos ejemplos.

**Reply by SujeiRodriguez** [reply]
gracias :D... de cuando pasa el mouse, intente con hover pero no le encontré bien
